examples/app-python/README.md

```diff
@@ -0,0 +1,39 @@
+# Example Python Payload Application
+
+This directory contains an example Payload Application using the Python.
+This is not production-ready code, but it is useful for learning how the platform functions.
+
+## Quickstart
+
+In the Antaris Cloud Platform, create a TrueTwin Satellite with a remote payload and download the associated config.
+Place that downloaded zip file in this directory.
+
+Build the app using the following command:
+
+```
+docker build --platform=linux/amd64 -t satos-payload-example-app-python .
+```
+
+Next, we can run the application in a container. The command below assumes that `CONFIG` is set to the name of the downloaded file in your current working directory:
+
+```
+docker run --platform=linux/amd64 -e CONFIG=$CONFIG -v $(pwd):/workspace -it satos-payload-example-app-python 
```

Comment:
I gave CONFIG as absolute oath of .zip file. But I got following error :

**$ export CONFIG=~/Downloads/_Packages_upload_RahulPayload-1684750606-aaa28e2d01.zip 
rahul@Rahul:~/projects/payload_git/test/SatOS-Payload-SDK/examples/app-python$ docker run --platform=linux/amd64 -e CONFIG=$CONFIG -v $(pwd):/workspace -it satos-payload-example-app-python
unzip:  cannot find or open /workspace//home/rahul/Downloads/_Packages_upload_RahulPayload-1684750606-aaa28e2d01.zip, /workspace//home/rahul/Downloads/_Packages_upload_RahulPayload-1684750606-aaa28e2d01.zip.zip or /workspace//home/rahul/Downloads/_Packages_upload_RahulPayload-1684750606-aaa28e2d01.zip.ZIP.**

In root-entrypoint.sh, CONFIG_ABS is calculated appending $pwd. 
So, we may need to specify explicitly about 'give relative path w.r.t. current dir' .

Reply:
Yes, that is right. I have fixed this in the docs.